src/pranitaPJun20/BankingSystem.java

```diff
@@ -0,0 +1,87 @@
+package pranitaPJun20;
+
+public class BankingSystem {
+//Instance Variables
+	int debitCount;
+	int creditCount;
+	int printBalanceCount;
+	int balance;
+//Static variables
+	static int staticDebitCount;
+	static int staticCreditCount;
+	static int staticPrintBalCount;
+	
+//Method to Debit amount
+		void debitAmnt(int amountDebited ){
+			debitCount= debitCount+1;
+			//debitCount++ ; //we can use this also
+			//debitCount+=1 ;//we can use this command also	
+			staticDebitCount=staticDebitCount+1;
+			balance= balance-amountDebited;
+				if(balance<=100) {
```

Comment:
This is not the condition that we expected. Please review https://github.com/KrishnaKTechnocredits/JAVATechnoJun20/pull/100